releases/release-1.31/release-notes/maps/pr-126205-map.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 126205
+releasenote:
+  text: Promoted _split node image filesystem_ support. The `KubeletSeparateDiskGC` feature
```

Comment:
`SplitImageFilesystem` or `KubeletSeparateDiskGC` (or both?)

Reply:
@kwilczynski both presumably?

Reply:
I think the feature going into beta is called `KubeletSeparateDiskGC`. https://github.com/kubernetes/kubernetes/pull/126205/files

Reply:
@npolshakova I find it confusing that the title and links in the pr refer to KEP-4191 Split Image Filesystem promotion to Beta 🤔 but you are right the actual change in the PR seems to be for KubeletSeparateDiskGC.

Reply:
```suggestion
  text: The `KubeletSeparateDiskGC` feature gate is now beta. This split image filesystem feature enables kubelet to perform garbage collection of images (read-only layers) and/or containers (writeable layers) deployed on separate filesystems.
```